docs/building-on-etherlink/transactions.md

```diff
@@ -273,6 +273,45 @@ run();
 
 ```
 
+## Getting instant confirmations
+
+Beginning with EVM node 0.48 and the version 6.0 upgrade, Etherlink supports instant confirmations.
+You can send a transaction with the `eth_sendRawTransactionSync` method and receive an instant confirmation from the sequencer that it intends to put the transaction in the next block, plus basic information about the completed transaction, such as its gas cost.
```

Comment:
```suggestion
You can send a transaction with the `eth_sendRawTransactionSync` method and receive an instant confirmation from the sequencer that it intends to put the transaction in the next block, plus his transaction receipt fully completed except block hash
```
or something like that, I think it's important to mention that we give a "receipt" because this a well-known data format in the ethereum community builder imo.